src/extension.ts

```diff
@@ -247,3 +249,41 @@ function registerCommands(context: ExtensionContext) {
     });
     context.subscriptions.push(restartServer);
 }
+
+function configureLanguage() {
+    languages.setLanguageConfiguration('rust', {
```

Comment:
Since `setLanguageConfiguration()` returns an `Disposable`, I think we should to register it as a disposable, e.g. what [official typescript extension](https://github.com/Microsoft/vscode/blob/master/extensions/typescript/src/typescriptMain.ts#L136) does.

Reply:
Good call. I fixed it.